src/AutoRest.CSharp/Common/Output/Builders/ClientBuilder.cs

```diff
@@ -67,7 +66,7 @@ public static string GetClientPrefix(string name, BuildContext context)
                 foreach (var request in operation.Requests)
                 {
                     RestClientMethod startMethod = restClient.GetOperationMethod(request);
-                    var name = nameOverrider?.Invoke(operationGroup, operation, startMethod) ?? operation.CSharpName();
+                    var name = operation.CSharpName();
```

Comment:
Has this functionality moved to another place?

Reply:
No, `nameOverrider` is always `null`.